site/content/blog/carvelizing-helm-chart.md

```diff
@@ -0,0 +1,293 @@
+---
+title: "Carvelizing Helm Chart"
+slug: carvelize-helm-chart
+date: 2022-02-16
+author: Rohit Aggarwal
+excerpt: "Use Carvel to author and consume helm chart "
+image: /img/logo.svg
+tags: ['carvel', 'helm', 'gitops']
+---
+
+In this blog post we will first show you how to wrap and distribute [`Bitnami nginx helm chart`](https://github.com/bitnami/charts/tree/master/bitnami/nginx) as a Carvel package, and then install it on the Kubernetes cluster via PackageInstall CR (via kapp-controller).
+
+## Why should I choose Carvel
+
+Kubernetes configuration takes many forms – plain YAML configurations, Helm charts, ytt templates, jsonnet templates, etc. Software running on Kubernetes lives in many different places: a Git repository, an archive over HTTP, a Helm repository, etc.
+
+Kapp-controller, a Carvel tool, provides software authors flexibility to choose their own configuration tools, while providing software consumers with consistent declarative APIs to customize, install, and update software on Kubernetes from various sources.
+
+## Prerequisites
+
+Basic knowledge of imgpkg, kbld, kapp-controller, kapp
+
+[`imgpkg`](https://carvel.dev/imgpkg/): A tool to package, distribute, and relocate your Kubernetes configuration and dependent OCI images as one OCI artifact: a bundle.
+
+[`kbld`](https://carvel.dev/kbld/): kbld incorporates image building and image pushing into your development and deployment workflows.
+
+[`kapp-controller`](https://carvel.dev/kapp-controller/): kapp-controller provides declarative APIs to create, customize, install, and update your Kubernetes applications into packages.
+
+[`kapp`](https://carvel.dev/kapp/): kapp is a cli used to deploy and view groups of Kubernetes resources as “application”.
+
+## Installing Carvel Tools
+
+We'll be using Carvel tools throughout this tutorial, so first we'll install them using `install.sh` script.
+
+```bash
+$ wget -O- https://carvel.dev/install.sh > install.sh
+
+# Inspect install.sh before running...
+$ sudo bash install.sh
+# Install kapp-controller
+$ kubectl apply -f https://github.com/vmware-tanzu/carvel-kapp-controller/releases/latest/download/release.yml
+```
+
+--------------------------
+
+## Authoring a Carvel Package
+
+To create a package, we need to create two CRs. We will go through step by step to author an nginx helm chart:
+
+**1. Create PackageMetadata CR**: Package Metadata contains very high level information and description about the package. Multiple versions of a package share same package metadata.
```

Comment:
nit:  remove the word "very"   --  there's some general advice about technical writing not needing adjectives like this,  even though we use them when speaking (and my early drafts always have them, too).

Reply:
Adopted